runtime/binding/python/py/decoder.py

```diff
@@ -49,6 +69,61 @@ def reset(self):
         """ Reset status for next decoding """
         _wenet.wenet_reset(self.d)
 
+    def get_model_by_lang(self, lang: str):
+
+        assert lang in ['chs', 'en']
+        # NOTE(Mddct): model_dir structure
+        # Path.Home()/.went
+        # - chs
+        #    - 20220506_u2pp_conformer_libtorch
```

Comment:
we can just ignore the dirname `20220506_u2pp_conformer_libtorch` since we only have one model  for each language.

And please notes the files in the download dir are:
final.zip
units.txt
We have renamed words.txt to units.txt recently.

Reply:
It's better if we decouple the download logic to another module.